### PR TITLE
add performance monitoring function

### PR DESCRIPTION
**Pull request by milivojm**
Function that will assist with CntoDev/cnto-assets#9

### CODE REVIEW

**Comment by freghar** [comment]
How about creating a `functions` addon and putting these things into it? ... I know there are plans to do similar things with some `Useful-Scripts` and I wanted to submit one for inclusion, but it ended up not that useful, https://github.com/CntoDev/Useful-Scripts/issues/3#issuecomment-770232727 .

The `functions` addon could work similar to https://github.com/freghar/arma-additions/tree/master/addons/functions, with folders for categories (and READMEs), but a shared `cnto` tag/namespace (see `config.cpp`).

**Comment by milivojm** [comment]
I did that before. I'm ok with that. @Seb105 , what do you think?

**Comment by freghar** [comment]
There's still that missing EOF newline in `config.cpp` and missing leading `\` in `cnto\additions\functions`, if that's something you want to fix. Otherwise looking good, assuming the function has other uses that `diag_codePerformance` doesn't cover.

**Review by freghar — APPROVED** [review]

File: addons/performance/fn_runTest.sqf [hunk 1]
```diff
@@ -0,0 +1,66 @@
+scriptName "cnto_performance_fnc_runTest";
+/* ----------------------------------------------------------------------------
+Function: cnto_performance_fnc_runTest
+Description:
+    Runs a test specified in _code for several times and logs average FPS and 
+    time in the logs with CNTO_Test_Start and CNTO_Test_End tags for easier
+    reporting. Needs to be run in scheduled environment.
```
[the remark this answers is not in the document]
Comment (milivojm) [reply addons/performance/fn_runTest.sqf]: Damn it. Didn't know about that function. Will wrap everything inside it ... what we need are just some fixed rpt tags for later analysis.

[the remark this answers is not in the document]
Comment (milivojm) [reply addons/performance/fn_runTest.sqf]: Hmm, wall. Brick wall here.

> The command will return as soon as possible if the tested code is slow and the duration of the command exceeds 1 second.

**Pull request merged**